src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -33,8 +33,11 @@ if `a` is a square, otherwise it is `-1`.
 
 This only deserves the name "character" when it is multiplicative,
 e.g., when `α` is a finite field. See `quadratic_char_mul`.
+
+We will later define `quadratic_char` to be a multiplicative character
+of type `mul_char F ℤ`, when the domain is a finite field `F`.
 -/
-def quadratic_char (α : Type*) [monoid_with_zero α] [decidable_eq α]
+def quadratic_char_def (α : Type*) [monoid_with_zero α] [decidable_eq α]
```

Comment:
We often use the suffix `_fun` (instead of `_def`) for the underlying function of a bundled object (such as a character, or a morphism).

Reply:
Riccardo Brasca said in an earlier comment:
> It's not so important, but I think we usually call these things `..._def`.
(instead of `quadratic_char'`).
But I have no problem renaming it to `..._fun`.